build.gradle

```diff
@@ -176,16 +173,6 @@ test {
     }
 }
 
-testScoverage {
```

Comment:
`test` configuration will now run scoverage whenever task is invoked: https://github.com/scoverage/gradle-scoverage#migration-to-3x